### PR TITLE
Secure URLs by replacing insecure aliases

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libdiscid.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
 class Libdiscid < Formula
   desc "C library for creating MusicBrainz and freedb disc IDs"
   homepage "https://musicbrainz.org/doc/libdiscid"
-  url "http://ftp.musicbrainz.org/pub/musicbrainz/libdiscid/libdiscid-0.6.2.tar.gz"
+  # Insecure, canonical alias:
+  # http://ftp.musicbrainz.org/pub/musicbrainz/libdiscid/
+  url "https://ftp.osuosl.org/pub/musicbrainz/libdiscid/libdiscid-0.6.2.tar.gz"
```
Comment (SMillerDev) [inline Formula/libdiscid.rb:6]: I don't think we should change URLS like this

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Comment by SMillerDev** [comment]
I don't think we should be changing away from the sources upstream provides because they don't use HTTPS. An issue should be made upstream and it should be made available instead.

**Comment by fxcoudert** [comment]
Neither do I. We use upstream's method of distribution, and we provide some security by checksum. If primary distribution is HTTP (or FTP), I'm adding an HTTPS mirror is actually more secure.

**Comment by vszakats** [comment]
Thanks for chiming in! As for notifying upstream I have a success rate (meaning an answer at all) of <10%, so I personally don't plan to pursue this, but adding the source/original URL as a mirror (or vice versa) seems like a good compromise. I started to look into this pattern for projects above 100000 downloads / year, but there are likely more cases and about 250 URLs with unmatched certificates in total.

There are about 20 GitHub-hosted URLs which could be upgraded to HTTPS with the necessary configuration, and I'm really unsure how to entice projects to do this, though it'd certainly a win for everybody.

I'm going to rework this PR.

**Comment by vszakats** [comment]
Reworked PR: https://github.com/Homebrew/homebrew-core/pull/63669

**Comment by vszakats** [comment]
FWIW, here's the list of cert name mismatches (haven't checked for `subjetAltName` in this pass):
```
https://0xcc.net/tty/tty/matrix.tty | CN=*.sakura.ne.jp
https://0xcc.net/ttyrec/ | CN=*.sakura.ne.jp
https://0xcc.net/ttyrec/ttyrec-1.0.8.tar.gz | CN=*.sakura.ne.jp
https://alice.loria.fr/software/geogram/doc/html/index.html | CN=annet2020.loria.fr
https://alpine.x10host.com/alpine/release/ | CN=x12.x10hosting.com
https://alpine.x10host.com/alpine/release/src/alpine-2.24.tar.xz | CN=x12.x10hosting.com
https://anjuta.org/ | CN=*.web-hosting.com
https://arblib.org/ | CN=*.webfaction.com
https://bugs.linuxnetworks.de/index.php?do=details&id=40 | CN=*.kasserver.com
https://catb.org/gpsd/ | CN=www.animadance.org
https://catb.org/~esr/intercal/ | CN=www.animadance.org
https://catb.org/~esr/intercal/intercal-0.31.tar.gz | CN=www.animadance.org
https://catb.org/~esr/ski/ | CN=www.animadance.org
https://cgns.org/ | CN=shortener.secureserver.net
https://chewing.im/ | CN=*.github.com
https://coccinelle.lip6.fr/ | CN=pages2.lip6.fr
https://coconut-lang.org/ | CN=*.github.com
https://colorcode.laebisch.com/ | CN=beilzeit.de
https://colorcode.laebisch.com/download/ColorCode-0.8.5.tar.gz | CN=beilzeit.de
https://cristal.inria.fr/~fpottier/menhir | CN=pauillac.paris.inria.fr
https://czmq.zeromq.org/ | CN=*.wikidot.com
https://digit-labs.org/files/tools/synscan/ | CN=www.ev6.net
https://digit-labs.org/files/tools/synscan/releases/synscan-5.02.tar.gz | CN=www.ev6.net
https://dist.schmorp.de/libev/ | CN=www.porttracker.com
https://dist.schmorp.de/libev/Attic/libev-4.33.tar.gz | CN=www.porttracker.com
https://dist.schmorp.de/liblzf/liblzf-3.6.tar.gz | CN=www.porttracker.com
https://dist.schmorp.de/rxvt-unicode/rxvt-unicode-9.22.tar.bz2 | CN=www.porttracker.com
https://dlib.net/ | CN=*.prod.iad2.secureserver.net
https://dlib.net/files/dlib-19.21.tar.bz2 | CN=*.prod.iad2.secureserver.net
https://download.camlcity.org/download/ | CN=gitlab.camlcity.org
https://download.camlcity.org/download/findlib-1.8.1.tar.gz | CN=gitlab.camlcity.org
https://download.gopivotal.com/rest-shell/1.2.1/rest-shell-1.2.1.RELEASE.tar.gz | CN=*.cloudfront.net
https://elinks.cz/elinks.git | CN=delmak.bofh.cz
https://elinks.or.cz/ | CN=delmak.bofh.cz
https://elinks.or.cz/download/elinks-0.11.7.tar.bz2 | CN=delmak.bofh.cz
https://entrproject.org/ | CN=eradman.com
https://entrproject.org/code/entr-4.6.tar.gz | CN=eradman.com
https://flintlib.org/ | CN=*.webfaction.com
https://flintlib.org/flint-2.6.3.tar.gz | CN=*.webfaction.com
https://fmv.jku.at/picosat/picosat-960.tar.gz | CN=s05zid04-web.zid.jku.at
https://ftp.alsa-project.org/pub/lib/alsa-lib-1.2.1.2.tar.bz2 | CN=alsa-project.org
https://ftp.cs.toronto.edu/cs/ftp/pub/hugh/jove-dev/jove4.16.0.73.tgz | CN=www.cs.toronto.edu
https://ftp.indexdata.dk/pub/metaproxy/metaproxy-1.19.1.tar.gz | CN=*.indexdata.com
https://ftp.indexdata.dk/pub/pazpar2/ | CN=*.indexdata.com
https://ftp.indexdata.dk/pub/pazpar2/pazpar2-1.14.0.tar.gz | CN=*.indexdata.com
https://ftp.indexdata.dk/pub/yaz/yaz-5.30.3.tar.gz | CN=*.indexdata.com
https://ftp.indexdata.dk/pub/yazpp/yazpp-1.7.1.tar.gz | CN=*.indexdata.com
https://ftp.indexdata.dk/pub/zebra/idzebra-2.2.0.tar.gz | CN=*.indexdata.com
https://ftp.midnight-commander.org/ | CN=*.osuosl.org
https://ftp.musicbrainz.org/pub/musicbrainz/libdiscid/ | CN=*.osuosl.org
https://ftp.musicbrainz.org/pub/musicbrainz/libdiscid/libdiscid-0.6.2.tar.gz | CN=*.osuosl.org
https://ftp.parrot.org/releases/supported/8.1.0/parrot-8.1.0.tar.bz2 | CN=*.osuosl.org
https://ftp.rpm.org/popt/releases/popt-1.x/ | CN=*.osuosl.org
https://ftp.rpm.org/popt/releases/popt-1.x/popt-1.18.tar.gz | CN=*.osuosl.org
https://ftp.rpm.org/releases/rpm-4.15.x/rpm-4.15.1.tar.bz2 | CN=*.osuosl.org
https://fuse.superglue.se/aamath/ | CN=www.crossbones.se
https://fuse.superglue.se/aamath/aamath-0.3.tar.gz | CN=www.crossbones.se
https://fy.chalmers.se/~appro/linux/DVD+RW/ | CN=w3hotell.portal.chalmers.se
https://fy.chalmers.se/~appro/linux/DVD+RW/tools/dvd+rw-tools-7.1.tar.gz | CN=w3hotell.portal.chalmers.se
https://gearman.org/ | CN=*.github.com
https://genometools.org/ | CN=zenlicensemanager.com
https://geoserver.org/ | CN=*.github.com
https://gkrellm.srcbox.net/releases/ | CN=box.srcbox.net
https://gkrellm.srcbox.net/releases/gkrellm-2.3.11.tar.bz2 | CN=box.srcbox.net
https://gva.noekeon.org/blahtexml/ | CN=mail.noekeon.org
https://gva.noekeon.org/blahtexml/blahtexml-0.9-src.tar.gz | CN=mail.noekeon.org
https://hevea.inria.fr/ | CN=pauillac.paris.inria.fr
https://hevea.inria.fr/old/ | CN=pauillac.paris.inria.fr
https://hevea.inria.fr/old/hevea-2.34.tar.gz | CN=pauillac.paris.inria.fr
https://iolanguage.com/ | CN=*.nfshost.com
https://jemalloc.net/ | CN=www.github.com
https://jim.tcl.tk/index.html | CN=sqlite.org
https://jonls.dk/redshift/ | CN=*.cloudfront.net
https://libccd.danfis.cz/ | CN=*.securenet.cz
https://libcec.pulse-eight.com/ | CN=www.pulse-eight.com
https://libdill.org/ | CN=*.github.com
https://liblouis.org/ | CN=*.github.com
https://libmill.org/ | CN=*.github.com
https://libmill.org/libmill-1.18.tar.gz | CN=*.github.com
https://mallet.cs.umass.edu/dist/mallet-2.0.8.tar.gz | CN=mantis.cs.umass.edu
https://mattmahoney.net/dc/calgarytest2.zpaq | CN=*.secure.hostingprod.com
https://mattmahoney.net/dc/zpaq.html | CN=*.secure.hostingprod.com
https://mattmahoney.net/dc/zpaq715.zip | CN=*.secure.hostingprod.com
https://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/cminch.ttf | CN=*.dante.de
https://mirrors.ctan.org/web/noweb.zip | CN=*.dante.de
https://mirrors.jenkins-ci.org/war-stable/ | CN=pkg.origin.jenkins.io
https://mirrors.jenkins.io/war-stable/2.249.2/jenkins.war | CN=pkg.origin.jenkins.io
https://mirrors.jenkins.io/war/2.264/jenkins.war | CN=pkg.origin.jenkins.io
https://mpir.org/ | CN=*.webfaction.com
https://mpir.org/mpir-3.0.0.tar.bz2 | CN=*.webfaction.com
https://nekohako.xware.cx/id3tool/ | CN=chris.collins.id.au
https://nekohako.xware.cx/id3tool/id3tool-1.2a.tar.gz | CN=chris.collins.id.au
https://nlp.ist.i.kyoto-u.ac.jp/EN/index.php?JUMAN%2B%2B | CN=*.sakura.ne.jp
https://nlp.ist.i.kyoto-u.ac.jp/index.php?JUMAN | CN=*.sakura.ne.jp
https://nlp.ist.i.kyoto-u.ac.jp/nl-resource/juman/juman-7.01.tar.bz2 | CN=*.sakura.ne.jp
https://oldhome.schmorp.de/marc/data/ | CN=www.porttracker.com
https://oldhome.schmorp.de/marc/data/fcrackzip-1.0.tar.gz | CN=www.porttracker.com
https://oldhome.schmorp.de/marc/fcrackzip.html | CN=www.porttracker.com
https://oldhome.schmorp.de/marc/liblzf.html | CN=www.porttracker.com
https://opencoarrays.org/ | CN=*.weebly.com
https://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/CMU-1-Small-Region.svs | CN=*.cmusatyalab.org
https://opentsdb.net/ | CN=*.github.com
https://pauillac.inria.fr/~ddr/ledit/ | CN=pauillac.paris.inria.fr
https://pauillac.inria.fr/~ddr/ledit/distrib/src/ledit-2.04.tgz | CN=pauillac.paris.inria.fr
https://phoenix.owl.de/tags/vasm1_8i.tar.gz | CN=ludwig.owl.de
https://picat-lang.org/ | CN=*.securedata.net
https://picat-lang.org/download/picat28_6_src.tar.gz | CN=*.securedata.net
https://point-at-infinity.org/ssss/ | CN=0pointer.net
https://point-at-infinity.org/ssss/ssss-0.5.tar.gz | CN=0pointer.net
https://pow.cx/ | CN=*.github.com
https://project-osrm.org/ | CN=*.github.com
https://projects.camlcity.org/projects/findlib.html | CN=gitlab.camlcity.org
https://projects.camlcity.org/projects/omake.html | CN=gitlab.camlcity.org
https://releases.0x539.de/gobby/gobby-0.5.0.tar.gz | CN=hosting.0x539.de
https://releases.0x539.de/libinfinity/ | CN=hosting.0x539.de
https://releases.0x539.de/libinfinity/libinfinity-0.6.8.tar.gz | CN=hosting.0x539.de
https://rigaux.org/hexedit.html | CN=www.zarb.org
https://sbuild.org/ | CN=*.github.com
https://sbuild.org/uploads/sbuild/0.7.7/sbuild-0.7.7-dist.zip | CN=*.github.com
https://sc68.atari.org/project.html | CN=admin.atari.org
https://sisc-scheme.org/ | CN=ephemeralfic.org
https://skinny-framework.org/ | CN=*.github.com
https://slashem.cvs.sourceforge.net/viewvc/slashem/slashem/configure?r1=1.13&r2=1.14&view=patch | CN=sourceforge.net
https://software.schmorp.de/pkg/libev.html | CN=www.porttracker.com
https://software.schmorp.de/pkg/rxvt-unicode.html | CN=www.porttracker.com
https://sqlmap.org/ | CN=*.github.com
https://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.gz | CN=distributions.freedesktop.org
https://tjaberg.com/ | CN=*.greengeeks.net
https://tjaberg.com/gsar121.zip | CN=*.greengeeks.net
https://todotxt.org/ | CN=*.github.com
https://vroom-project.org/ | CN=verso-optim.com
https://websocketd.com/ | CN=*.github.com
https://wiremock.org/docs/running-standalone/ | CN=*.github.com
https://www.arpoison.net/ | CN=*.accountservergroup.com
https://www.arpoison.net/arpoison-0.7.tar.gz | CN=*.accountservergroup.com
https://www.catb.org/esr/reposurgeon/ | CN=www.animadance.org
https://www.catb.org/~esr/ascii/ | CN=www.animadance.org
https://www.catb.org/~esr/ascii/ascii-3.18.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/cvs-fast-export/ | CN=www.animadance.org
https://www.catb.org/~esr/cvs-fast-export/cvs-fast-export-1.55.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/cvsps/ | CN=www.animadance.org
https://www.catb.org/~esr/cvsps/cvsps-3.13.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/deheader | CN=www.animadance.org
https://www.catb.org/~esr/deheader/deheader-1.7.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/gif2png/ | CN=www.animadance.org
https://www.catb.org/~esr/gif2png/gif2png-2.5.13.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/greed/ | CN=www.animadance.org
https://www.catb.org/~esr/greed/greed-4.2.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/morse/ | CN=www.animadance.org
https://www.catb.org/~esr/morse/morse-2.5.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/ski/ski-6.13.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/src/ | CN=www.animadance.org
https://www.catb.org/~esr/src/src-1.28.tar.gz | CN=www.animadance.org
https://www.catb.org/~esr/wumpus/ | CN=www.animadance.org
https://www.catb.org/~esr/wumpus/wumpus-1.7.tar.gz | CN=www.animadance.org
https://www.ch-werner.de/sqliteodbc/ | CN=*.homepage.t-online.de
https://www.convertlit.com/ | CN=recesja.icm.edu.pl
https://www.convertlit.com/clit18src.zip | CN=recesja.icm.edu.pl
https://www.darkice.org/ | CN=*.syndicat.com
https://www.edge-security.com/theharvester.php | CN=*.aruba.it
https://www.feynarts.de/cuba/ | CN=wwwth.mpp.mpg.de
https://www.feynarts.de/cuba/Cuba-4.2.tar.gz | CN=wwwth.mpp.mpg.de
https://www.fpx.de/fp/Software/UUDeview/ | CN=000aaa.webhosting.dsinet.de
https://www.fpx.de/fp/Software/UUDeview/download/uudeview-0.5.20.tar.gz | CN=000aaa.webhosting.dsinet.de
https://www.ghostgum.com.au/software/epstool.htm | CN=*.ssllock.net
https://www.gnuplot.info/ | CN=prod.vtti.vt.edu
https://www.gnuplot.info/ | CN=prod.vtti.vt.edu
https://www.gobosoft.com/ | CN=cluster021.hosting.ovh.net
https://www.gprolog.org/ | CN=*.univ-paris1.fr
https://www.gprolog.org/gprolog-1.4.5.tar.gz | CN=*.univ-paris1.fr
https://www.greenwoodsoftware.com/less/index.html | CN=rack13.fast-web-servers.com
https://www.greenwoodsoftware.com/less/less-551.tar.gz | CN=rack13.fast-web-servers.com
https://www.jedrea.com/chmlib | CN=*.nfshost.com
https://www.jedrea.com/chmlib/chmlib-0.40.tar.gz | CN=*.nfshost.com
https://www.jsonschema2pojo.org/ | CN=*.herokuapp.com
https://www.kfish.org/software/tractorgen/ | CN=horms.net
https://www.kfish.org/software/tractorgen/dl/tractorgen-0.31.7.tar.gz | CN=horms.net
https://www.kite-language.org/ | CN=voldemort.lifeafterking.org
https://www.kite-language.org/files/kite-1.0.4.tar.gz | CN=voldemort.lifeafterking.org
https://www.knossos.net.nz/downloads/dterm-0.5.tgz | CN=*.on.net.nz
https://www.knossos.net.nz/resources/free-software/dterm/ | CN=*.on.net.nz
https://www.laffeycomputer.com/rpl.html | CN=tidyupgrade.com
https://www.laffeycomputer.com/shellutils.html | CN=tidyupgrade.com
https://www.laffeycomputer.com/whatmask.html | CN=tidyupgrade.com
https://www.lcdproc.org/ | CN=smtp.omnipotent.net
https://www.lichteblau.com/download/ldapvi-1.7.tar.gz | CN=www.askja.de
https://www.lichteblau.com/ldapvi/ | CN=www.askja.de
https://www.martiansoftware.com/nailgun/ | CN=*.accountservergroup.com
https://www.mathematik.uni-ulm.de/users/ag/yaze-ag/ | CN=anmelden.mathematik.uni-ulm.de
https://www.mathematik.uni-ulm.de/users/ag/yaze-ag/devel/yaze-ag-2.40.5_with_keytrans.tar.gz | CN=anmelden.mathematik.uni-ulm.de
https://www.mcternan.me.uk/mscgen/ | CN=*.gridhost.co.uk
https://www.mcternan.me.uk/mscgen/software/mscgen-src-0.20.tar.gz | CN=*.gridhost.co.uk
https://www.meta-environment.org/releases/asc-support-2.6.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/asf-support-1.8.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/aterm-2.8.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/c-library-1.2.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/config-support-1.4.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/error-support-1.6.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/pandora-1.6.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/pgen-2.8.1.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/pt-support-2.4.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/ptable-support-1.2.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/rstore-support-1.0.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/sdf-2.6.3.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/sdf-support-2.5.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/sglr-4.5.3.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/tide-support-1.3.1.tar.gz | CN=update.rascal-mpl.org
https://www.meta-environment.org/releases/toolbuslib-1.1.tar.gz | CN=update.rascal-mpl.org
https://www.mutt.org/ | CN=*.cass.oregonstate.edu
https://www.newlisp.org/ | CN=*.nfshost.com
https://www.newlisp.org/downloads/newlisp-10.7.5.tgz | CN=*.nfshost.com
https://www.philippwesche.org/200811/whohas/intro.html | CN=*.tralios.de
https://www.qhull.org/ | CN=*.securedata.net
https://www.qhull.org/download/qhull-2020-src-8.0.0.tgz | CN=*.securedata.net
https://www.ravenports.com/catalog/bucket_2D/readline/standard/ | CN=sting.dragonflybsd.org
https://www.rqlite.com/ | CN=www.philipotoole.com
https://www.sane-project.org/ | CN=*.gitlab.io
https://www.scalastyle.org/command-line.html | CN=*.nfshost.com
https://www.squid-cache.org/ | CN=*.marwan.ma
https://www.squid-cache.org/Versions/v4/ | CN=*.marwan.ma
https://www.squid-cache.org/Versions/v4/squid-4.13.tar.xz | CN=*.marwan.ma
https://www.uzix.org/cgvg.html | CN=*.webfaction.com
https://www.uzix.org/cgvg/cgvg-1.6.3.tar.gz | CN=*.webfaction.com
https://www.vapoursynth.com/ | CN=sansa.asoshared.com
https://www.vapoursynth.com/ | CN=sansa.asoshared.com
https://www.vapoursynth.com/ | CN=sansa.asoshared.com
https://www.vapoursynth.com/ | CN=sansa.asoshared.com
https://www.vapoursynth.com/doc/plugins.html | CN=sansa.asoshared.com
https://www.videlibri.de/xidel.html | CN=secureprojects.sourceforge.net
https://www.viewvc.org/ | CN=red-bean.com
https://www.winfield.demon.nl/ | CN=*.xs4all.nl
https://www.winfield.demon.nl/linux/ | CN=*.xs4all.nl
https://www.winfield.demon.nl/linux/antiword-0.37.tar.gz | CN=*.xs4all.nl
https://www.wiredtiger.com/ | CN=a248.e.akamai.net
https://www.xunsearch.com/scws/down/scws-1.2.3.tar.bz2 | CN=haimanchajian.com
https://www.zezula.net/en/mpq/stormlib.html | CN=leda.zarea.net
https://www.zint.org.uk/ | CN=*.robsroost.me.uk
https://x3270.bgp.nu/ | CN=bgp.nu
https://x3270.bgp.nu/download.html | CN=bgp.nu
https://x3270.bgp.nu/download/04.00/suite3270-4.0ga12-src.tgz | CN=bgp.nu
https://xmlsoft.org/ | CN=libvirt.org
https://xmlsoft.org/sources | CN=libvirt.org
https://xmlsoft.org/sources/ | CN=libvirt.org
https://xmlsoft.org/sources/libxml2-2.9.10.tar.gz | CN=libvirt.org
https://xmlsoft.org/sources/libxslt-1.1.34.tar.gz | CN=libvirt.org
https://zanata.org/ | CN=*.github.com
https://zbackup.org/ | CN=*.github.com
https://zsync.moria.org.uk/ | CN=*.secure-secure.co.uk
https://zsync.moria.org.uk/download/zsync-0.6.2.tar.bz2 | CN=*.secure-secure.co.uk
```

**Pull request closed without merging**